app/js/data/pool.js

```diff
@@ -72,6 +74,70 @@ class Pool {
       this.spaces[slot].used = true;
     }
   }
+
+  // Re-generate pool spaces from loaded card data
+  regenerateFromCardData(cardSpaces) {
+    // Reset all entries to unused
+    this.contents.forEach(reservation => {
+      reservation.entry.used = false;
+    });
+
+    // For each card space (excluding FREE), find matching entry in contents and mark as used
+    let poolIndex = 0;
+    for (let cardIndex = 0; cardIndex < 25; cardIndex++) {
```

Comment:
```suggestion
    for (let cardIndex = 0, poolIndex = 0; cardIndex < 25; cardIndex++) {
```
block variable scoping reasons if the function ever gets added to in the future.